[2025-06-27T04:25:52.497+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T04:25:52.586+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.create_ga4_events_dataset scheduled__2025-03-12T05:00:00+00:00 [queued]>
[2025-06-27T04:25:52.613+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.create_ga4_events_dataset scheduled__2025-03-12T05:00:00+00:00 [queued]>
[2025-06-27T04:25:52.616+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T04:25:52.710+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-03-12 05:00:00+00:00
[2025-06-27T04:25:52.733+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1445) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T04:25:52.738+0000] {standard_task_runner.py:72} INFO - Started process 1479 to run task
[2025-06-27T04:25:52.762+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0', 'create_ga4_events_dataset', 'scheduled__2025-03-12T05:00:00+00:00', '--job-id', '355', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmp5vse050l']
[2025-06-27T04:25:52.807+0000] {standard_task_runner.py:105} INFO - Job 355: Subtask create_ga4_events_dataset
[2025-06-27T04:25:53.149+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0.create_ga4_events_dataset scheduled__2025-03-12T05:00:00+00:00 [running]> on host f79cc72a155d
[2025-06-27T04:25:53.540+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-12T05:00:00+00:00'
[2025-06-27T04:25:53.564+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T04:25:53.624+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T04:25:53.643+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T04:25:53.646+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events.
[2025-06-27T04:25:53.647+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T04:25:53.648+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events in project: sixth-topic-349709 
[2025-06-27T04:25:54.996+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T04:25:55.228+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T04:25:55.229+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0, task_id=create_ga4_events_dataset, run_id=scheduled__2025-03-12T05:00:00+00:00, execution_date=20250312T050000, start_date=20250627T042552, end_date=20250627T042555
[2025-06-27T04:25:55.460+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T04:25:55.584+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T04:25:55.596+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
